packages/tizen_rpc_port/example/client/lib/message_client.dart

```diff
@@ -6,7 +6,7 @@
 
 import 'package:tizen_rpc_port/tizen_rpc_port.dart';
 
-const String _tidlVersion = '1.9.1';
+const String _tidlVersion = '1.10.6';
```

Comment:
When is TIDLC 1.10.6 scheduled for release?

Reply:
As far as I know, there is a tizen studio release around March.
I think it can be reflected at that time. Apart from this, public release continues.